packages/frontend/src/functions/CursorWidget/BubbleWidget.ts

```diff
@@ -0,0 +1,61 @@
+import * as monaco from "monaco-editor";
+import reactMonaco from "../Monaco";
+import classes from "./Widget.module.css";
+
+const WIDGET_PREFIX = "bubble";
+
+/**
+ * A Monaco Content Widget that represents the "bubble" over another persons
+ * cursor
+ */
+class BubbleWidget implements monaco.editor.IContentWidget {
+  private userId: string;
+
+  private domNode: HTMLElement;
+
+  private position: monaco.IPosition = { column: 0, lineNumber: 0 };
+
+  /**
+   * @param userId This will be displayed inside the bubble
+   * @param color This will be the background color of the bubble
+   */
+  public constructor(userId: string, color = "#f44336") {
+    this.userId = userId;
+
+    this.domNode = document.createElement("div");
+    this.domNode.className = `${classes.widget} ${classes.bubble}`;
```

Comment:
```suggestion
    this.domNode.className = `${classes.widget}-${classes.bubble}`;
```

Reply:
The space is intended, there are two different classes that I want to apply